main.py

```diff
@@ -25,23 +25,24 @@
 'Always use a database. What\'s a JSON?',
 'Powered by Electricity',
 'Who still writes docs in 2018?',
-"First normal form? What does that mean?",
-"By using a relational database but with nonrelational practices we get the worst of both worlds!",
-"I haven\'t paid attention or read any comments, therefor it\'s impossible to understand!",
-"Don\'t use that! Oh, you\'re asking why? Well... just don\'t it.",
-"I don\'t wanna explain, just Google it.",
-"What are cogs?",
-"This is MY project. You\'re just freeloaders.",
-"You've got three weeks to fix EVERYTHING.",
-"No-one agrees? Too bad! My idea it is.",
-"The next version will be written in Java only!"
+'First normal form? What does that mean?',
+'By using a relational database with nonrelational practices we get the worst of both worlds!',
+'I haven\'t paid attention or read any comments, therefore it\'s impossible to understand!',
+'Don\'t use that! Oh, you\'re asking why? Well... just don\'t it.',
+'I don\'t wanna explain, just Google it.',
+'What are cogs?',
+'This is MY project. You\'re just freeloaders.',
+'You\'ve got three weeks to fix EVERYTHING.',
+'No-one agrees? Too bad! My idea it is.',
+'The next version will be written in Java only!'
```

Comment:
Why did you change all this?

Reply:
I just thought it would be more consistent, but it doesn't really affect anything